content/docs/concepts/architecture/index.md

```diff
@@ -50,11 +46,19 @@ The *mongodb-datastore* provides means to store event data in a MongoDB deployed
 
 The *configuration-service* is a Keptn core component and used to manage resources for Keptn project-related entities, i.e., project, stage, and service. To store the resources with version control, a Git repository is used that is mounted as a persistent volume. Besides, this service has the functionality to upload the Git repository to any Git-based service such as GitLab, GitHub, Bitbucket, etc.
 
-### Control Plane related Services
+### shipyard-controller
+
+The *shipyard-controller* is responsible for managing all Keptn-related entities, such as projects, stages and services and provides an HTTP API that allows to perform CRUD operations on them. 
+Another responsibility of this service is the control of task sequences defined in the shipyard file of a project by sending out `.triggered` events whenever a task within a task sequence should be executed. 
+It will then listen for incoming `.started` and `.finished` events and use them to proceed within the task sequence.
 
-This category of services reacts on Keptn events and perform: (1) initial tasks like creating a project or creating a service, (2) continuous delivery tasks like deploying or promoting a service, and (3) orchestrational tasks for automating operations.
+### Execution Plane Services
```

Comment:
Can you please change it to `## Execution Plane Services` to move it one level up.